enable-grafana-monitoring.md

```diff
@@ -0,0 +1,67 @@
+# Enabling Grafana Monitoring for SecureSign
+
+This guide provides the commands to deploy both the Grafana operator
+and a Grafana instance in OpenShift. It also adds a Prometheus Datasource
+and configures a dashboard for monitoring Sigstore components.
+
+Prerequisites
+1. Make sure you have the [oc command-line tool](https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html) installed.
+2. Ensure you are logged into your OpenShift cluster.
+
+Note: This guide assumes you are using OpenShift version 4.12 or greater.
+
+## Step 1: Installing the operator
+
+This installs the Grafana operator into the `sigstore-monitoring` namespace.
+```bash
+oc apply -k grafana/operator
+```
+
+Tip: Verify the installation by running `oc get pods -n sigstore-monitoring`.
+
+## Step 2: Creating a grafana instance
+
+This creates a Grafana instance for the operator. Make sure to allow some time for the Grafana operator to install.
+
+```bash
+oc apply -k grafana/instance
+```
+
+## Step 3: Configuring grafana resources
+
+Apply the necessary tokens and role bindings to the service account `grafana-serviceaccount` in the `sigstore-monitoring` namespace
+
+```bash
+oc apply -k grafana/resources
+```
+## Step 4: Retrieveing secrets
+
+Retrieve all necessary secrets from the OpenShift cluster and apply them to the `datasource.yaml` file found at `grafana/dashboards/datasource.yaml`.
+
+```bash
+export BEARER_TOKEN=$(oc -n sigstore-monitoring get secrets grafana-sa-token -o=jsonpath="{.data.token}" | base64 -d)
+export MYSQL_USER=$(oc -n trillian-system get secrets trillian-mysql -o=jsonpath="{.data.mysql-user}" | base64 -d)
+export MYSQL_PASSWORD=$(oc -n trillian-system get secrets trillian-mysql -o=jsonpath="{.data.mysql-password}" | base64 -d)
+export MYSQL_DATABASE=$(oc -n trillian-system get secrets trillian-mysql -o=jsonpath="{.data.mysql-database}" | base64 -d)
+```
+
+```bash
+envsubst < grafana/dashboards/datasource.yaml > grafana/dashboards/temp_datasource.yaml 
+mv grafana/dashboards/temp_datasource.yaml grafana/dashboards/datasource.yaml
+```
+## Step 5: Creating datasources & dashboards
+
+Finally, the datasources and dashboards can be created.
+
+```bash
+oc apply -k grafana/dashboards
+```
```

Comment:
Instead of lines #49,50 above, how about remove the kustomization file there and then:

```bash
envsubst < grafana/dashboards/datasource.yaml | oc apply -f -
oc apply -f grafana/dashboards/dashboard.yaml
```
I like this better bc as a developer you don't have those extra git changes to undo

Reply:
I'm going to merge this as/is and change that as a follow-up :)